libs/blocks/merch-card/merch-card.js

```diff
@@ -0,0 +1,205 @@
+    };
+    const cardFooterRow = el.querySelector('.consonant-CardFooter-row');
+    const secureTransactionSign = el.querySelector('h6[id^="secure-transaction"]');
+    if (secureTransactionSign) {
+      const standardWrapper = createTag('div', { class: 'standard-wrapper' });
+      const secureTransactionWrapper = decorateWithSecureTransactionSign(secureTransactionSign);
+      standardWrapper.append(secureTransactionWrapper, cardFooterRow);
+      cardFooter?.append(standardWrapper);
+      secureTransactionSign.remove();
+    }
+
+    const altCta = Array.from(el.querySelectorAll('div > div[data-align="center"][data-valign="middle"]')).filter((data) => data.textContent?.trim() === 'alt-cta');
+    const altCtaMetaData = altCta[0]?.parentElement?.nextElementSibling?.querySelectorAll('div > div');
+    const altCtaRegex = /href="([^"]*)"/g;
+    if (altCtaMetaData?.length === 2 && altCtaMetaData[1]?.innerHTML?.match(altCtaRegex)) {
+      const originalCtaButton = cardFooterRow.querySelector('.consonant-CardFooter-cell--right');
+      const altCtaButton = originalCtaButton.cloneNode(true);
+      const checkboxContainer = createCheckbox(altCtaMetaData[0]);
+
+      altCtaButton.innerHTML = altCtaMetaData[1].innerHTML;
+      altCtaButton.classList.add('button--inactive');
+      checkboxContainer.querySelector('input[type="checkbox"]').addEventListener('change', (event) => {
+        if (event.target.checked) {
+          toggleButtonActiveState(originalCtaButton, altCtaButton);
+        } else {
+          toggleButtonActiveState(altCtaButton, originalCtaButton);
+        }
+      });
+      cardFooterRow.append(altCtaButton);
+      cardFooter.prepend(checkboxContainer);
+      altCta[0].parentNode.remove();
+      altCtaMetaData[0].parentNode.remove();
+    }
+  };
+
+  decorateAlternativeCta();
+  cardFooter.querySelectorAll('.consonant-CardFooter-cell').forEach((cell) => cell.classList.add(`consonant-${cardType}-cell`));
+};
+
+const addInner = (el, cardType, merchCard) => {
+  const titles = Array.from(el.querySelectorAll('h1, h2, h3, h4, h5, h6'));
+  const rows = Array.from(el.querySelectorAll('p'));
+  const styles = Array.from(el.classList);
+  const merch = styles.includes('merch-card');
+  const pElement = merch && el.querySelector(':scope > div > div > p:last-of-type');
+  const links = pElement ? pElement.querySelectorAll('a') : el.querySelectorAll('a');
+
+  const inner = el.querySelector(':scope > div:not([class])');
+  inner.classList.add(`consonant-${cardType}-inner`);
+  const title = createTitle(titles, cardType);
+  const description = createDescription(rows, cardType, inner);
+
+  inner.prepend(title);
+  inner.append(description);
+  addFooter(links, inner, merchCard);
+  decorateFooter(el, cardType);
+  merchCard.append(inner);
+};
+
+const decorateRibbon = (el, cardType) => {
+  const ribbonMetadata = el.querySelectorAll('div > div[data-align="center"][data-valign="middle"]');
```

Comment:
These are added w/ some word authoring, Is this `div[data-align="center"][data-valign="middle"]` reliable?

Reply:
Why is this question dismissed? Can you provide context around the inquary?

Reply:
```suggestion
  const ribbonMetadata = el.querySelectorAll('div > div');
```

Reply:
@VKniaz  - I don't think we want to rely on authors to always have the ribbon metadata row formatted w/ these specific `data-align` attributes. These are generated from word formatting and may/will vary depending on who and how this is authored. I did check the suggested change above and it actually breaks cards that don't have this optional 'ribbon' row. So we'll probably want to get a more generic query and test for nested > div:children count. 
Ex.  
```
const firstRow = el.querySelectorAll(':scope > div');
// ... check if first row has two cells ☑️ (we know its a banner metadata-row) 
```

Reply:
See my test page, I just 'removed format' in the word styling and it breaks the first card. 
https://merch-card--milo--adobecom.hlx.page/drafts/rparrish/cards/merch-cards

Reply:
Here's an idea...

<img width="1681" alt="Screen Shot 2023-08-28 at 5 06 40 PM" src="https://github.com/adobecom/milo/assets/2086146/bb36bafb-c975-4ad6-b3a6-2b456fb987db">
<img width="1613" alt="Screen Shot 2023-08-28 at 5 06 50 PM" src="https://github.com/adobecom/milo/assets/2086146/a434a4e0-8c1f-42bb-91c2-09263bdadb63">